PWGHF/TableProducer/candidateCreatorLb.cxx

```diff
@@ -262,10 +254,13 @@ struct HfCandidateCreatorLbMc {
   Produces<aod::HfCandLbMcRec> rowMcMatchRec;
   Produces<aod::HfCandLbMcGen> rowMcMatchGen;
 
-  void process(aod::HfCandLb const& candidates,
-               aod::HfCand3Prong const&,
-               aod::TracksWMc const&,
-               aod::McParticles const& mcParticles)
+  /// @brief dummy process function, to be run on data
+  void process(aod::Tracks const&) {}
+
+  void processMC(aod::HfCandLb const& candidates,
```

Comment:
```suggestion
  void processMc(aod::HfCandLb const& candidates,
```

Reply:
Still in the task

Reply:
Sorry. I thought here your suggestion is to make the c lowercase, which is done. Should I remove the dummy function or what do you mean?

Reply:
You thought right. I suggested to make the c lowercase in the task.